crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -1718,7 +1736,9 @@ impl<'db> TypeInferenceBuilder<'db> {
 
             // Inference of bases deferred in stubs
             // TODO also defer stringified generic type parameters
```

Comment:
I think this TODO should at least be updated in light of the code below (sort of) handling this case now